src/index.js

```diff
@@ -0,0 +1,14 @@
+import './style.css';
+import { addList, displayList } from './modules/appbuilder.js';
+import localGet from './modules/localstorage';
+
+const addBtn = document.querySelector('.add');
+const task = document.querySelector('.task');
+
+window.addEventListener('load', () => {
+  displayList();
+});
```

Comment:
- [ ] Please, consider fixing this issue by displaying the tasks. Here in the screenshot, you can see, that the task title is not fully displaying, but in Local Storage it is ok.
![image](https://user-images.githubusercontent.com/49064106/194300201-a60ea480-c2c3-403d-9884-4f75c6fd79ae.png)

Reply:
> @elmar8287. Thanks for the review but I don't understand the change you want me to make in the local storage.

@eerapheal Here I want to show you, that the task titles are displaying not correctly. In the screen shot you can see that in your Local Storage the tasks titles are correctly displaying, aka Task 1, Task 2, Task 3, etc... but on the left side of the screenshot, on-page, in the list of tasks we can see only Task, Task, Task, etc... You do not have to make changes in your Local Storage, but you have to fix the displaying of your tasks correctly in the page. If you have any questions, we can meet via Zoom and explain in more detail. Happy coding 👍

Reply:
Hey,
I tried to contact you on zoom but I could't find your name. please send me your name so that I can contact you